en/orm/database-basics.rst

```diff
@@ -30,7 +30,7 @@ Supported Databases
 
 CakePHP supports the following relational database servers:
 
-* MySQL 5.1+
+* MySQL 5.5+
```

Comment:
MySQL 5.5.3+

Reply:
We support all of 5.5 if you use the right encoding.